--- /etc/gitlab/gitlab.rb.orig	2018-12-22 17:31:25.599644518 +0000
+++ /etc/gitlab/gitlab.rb	2018-12-22 17:43:04.115252738 +0000
@@ -1041,20 +1041,20 @@
 # nginx['server_names_hash_bucket_size'] = 64
 
 ### Nginx status
-# nginx['status'] = {
-#  "enable" => true,
-#  "listen_addresses" => ["127.0.0.1"],
-#  "fqdn" => "dev.example.com",
-#  "port" => 9999,
-#  "vts_enable" => true,
-#  "options" => {
-#    "stub_status" => "on", # Turn on stats
-#    "server_tokens" => "off", # Don't show the version of NGINX
-#    "access_log" => "off", # Disable logs for stats
-#    "allow" => "127.0.0.1", # Only allow access from localhost
-#    "deny" => "all" # Deny access to anyone else
-#  }
-# }
+nginx['status'] = {
+  "enable" => true,
+  "listen_addresses" => ["127.0.0.1"],
+  "fqdn" => "localhost",
+  "port" => 8060,
+  "vts_enable" => true,
+  "options" => {
+    "stub_status" => "on", # Turn on stats
+    "server_tokens" => "off", # Don't show the version of NGINX
+    "access_log" => "off", # Disable logs for stats
+    "allow" => "127.0.0.1", # Only allow access from localhost
+    "deny" => "all" # Deny access to anyone else
+  }
+}
 
 ################################################################################
 ## GitLab Logging
