--- gitlab.rb.orig	2020-11-08 15:47:07.886114149 +0000
+++ gitlab.rb	2020-11-08 16:01:08.737971127 +0000
@@ -1353,19 +1353,19 @@
 # nginx['request_buffering_off_path_regex'] = "/api/v\\d/jobs/\\d+/artifacts$|\\.git/git-receive-pack$|\\.git/gitlab-lfs/objects|\\.git/info/lfs/objects/batch$"
 
 ### Nginx status
-# nginx['status'] = {
-#  "enable" => true,
-#  "listen_addresses" => ["127.0.0.1"],
-#  "fqdn" => "dev.example.com",
-#  "port" => 9999,
-#  "vts_enable" => true,
-#  "options" => {
-#    "server_tokens" => "off", # Don't show the version of NGINX
-#    "access_log" => "off", # Disable logs for stats
-#    "allow" => "127.0.0.1", # Only allow access from localhost
-#    "deny" => "all" # Deny access to anyone else
-#  }
-# }
+nginx['status'] = {
+  "enable" => true,
+  "listen_addresses" => ["127.0.0.1"],
+  "fqdn" => "localhost",
+  "port" => 8060,
+  "vts_enable" => true,
+  "options" => {
+    "server_tokens" => "off", # Don't show the version of NGINX
+    "access_log" => "off", # Disable logs for stats
+    "allow" => "127.0.0.1", # Only allow access from localhost
+    "deny" => "all" # Deny access to anyone else
+  }
+}
 
 ################################################################################
 ## GitLab Logging
